pkg/controller/postgresql.go

```diff
@@ -312,7 +312,7 @@ func (c *Controller) processClusterEventsQueue(idx int, stopCh <-chan struct{},
 	for {
 		obj, err := c.clusterEventQueues[idx].Pop(cache.PopProcessFunc(func(interface{}) error { return nil }))
 		if err != nil {
-			if err == cache.FIFOClosedError {
+			if err == cache.ErrFIFOClosed {
```

Comment:
The change is caused by renaming in the `cache` package ; fixes a lint violation there [reference](https://github.com/kubernetes/client-go/commit/2b8d87c0821de4a8eaa8da496845d8a04cf1eadf#diff-c74ce1b4d1c0c7c6f63898649343d81bL37)